Makefile

```diff
@@ -15,13 +15,15 @@ $(DCURL_LIB): $(DCURL_DIR)
 	$(info Modify $^/build/local.mk for your environments.)
 	$(MAKE) -C $^ all
 
+MQTT: $(DCURL_LIB) $(MOSQITTO_LIB)
```

Comment:
Can you validate the build process for MQTT?

Reply:
We are using `make MQTT` in CI, now.